src/sentry/incidents/action_handlers.py

```diff
@@ -197,8 +196,6 @@ def generate_incident_trigger_email_context(
     user=None,
 ):
     trigger = alert_rule_trigger
-    incident_trigger = IncidentTrigger.objects.get(incident=incident, alert_rule_trigger=trigger)
```

Comment:
With the new changes, this `IncidentTrigger` doesn't necessarily exist, but it doesn't matter because we don't actually need it. We can access the `date_added` directly from the `Incident`